addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7763,7 +7763,17 @@ msgctxt "#13476"
 msgid "16 bit per channel"
 msgstr ""
 
-#empty strings from id 13477 to 13504
+#: system/settings/settings.xml
+msgctxt "#13477"
+msgid "Texture Compression"
```

Comment:
Commit name is misleading: commit has nothing to do with 10bit gui. It was left over previous commit.
Commit name should be

VideoPlayer: Add option to disable texture compression